src/Chapter2.hs

```diff
@@ -620,7 +636,9 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven [] = []
+takeEven (x:xs) = x : takeEven (drop 1 xs)
```

Comment:
That is a clever solution! 🧠 

Alternatively, you could pattern match on that as well, e.g. `(x: _ : xs) = x : takeEven xs`.
(Note that you would need to handle the case with one element as well in that case)>

Reply:
Oh this is beautiful ! Thanks ! Amazing language :-)